babel/messages/frontend.py

```diff
@@ -866,13 +1108,17 @@ class CommandLineInterface:
         'extract': 'extract messages from source files and generate a POT file',
         'init': 'create new message catalogs from a POT file',
         'update': 'update existing message catalogs from a POT file',
+        'concat': 'concatenates and merges the specified PO files',
+        'merge': 'combines two Uniforum-style PO files into one',
```

Comment:
I still have no idea what an Uniforum-style PO file is; do we not support whatever is a non-Uniforum-style PO file?